fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -20,7 +20,7 @@ SPIN_ECHO_METHOD_OPT="TOPUP"
 #  Load Function Libraries
 # --------------------------------------------------------------------------------
 
-source $HCPPIPEDIR_Global/log.shlib # Logging related functions
+source ${HCPPIPEDIR_Global}/log.shlib           # Logging related functions
```

Comment:
For simplicity and consistency let's source as 
`source $HCPPIPEDIR/global/scripts/log.shlib` (rather than using the obscure `HCPPIPEDIR_Global` variable.
(Although, sounds like this may be getting replaced anyway by a source of the debug.shlib).

Reply:
Yes. Replaced by debug.shlib.